str.lisp

```diff
@@ -135,10 +135,10 @@
                   string/char
                   (subseq s index)))))
 
-(defun split (separator s &key (omit-nulls *omit-nulls*))
+(defun split (separator s &key (omit-nulls *omit-nulls*) (limit nil))
```

Comment:
It can be `limit` alone, it is `nil` by default.

Reply:
Done!